### ISSUE TITLE
Support Windows Server non-Core VM's

### ISSUE THREAD

**Issue by JeremyRand** [cirruslabs/cirrus-ci-docs#982]
# Description

Currently, Cirrus's built-in Windows VM images only cover the `windows-2019-core-for-containers` Windows variant.  It would be nice if the `windows-2019` (not Core, and not for containers) variant was also supported.  (Installing Docker in such images is not required for me.)

It looks like it should be trivial to edit the JSON to do this, but as I don't have a Google Compute Engine account, testing this is nontrivial for me.

# Context

I'm trying to run CI tests on GUI software (Firefox) that doesn't run well in Windows Server Core (as it requires a GUI, and the Core variant is CLI-only).

**Comment by JeremyRand** [comment]
Ah, interesting, I hadn't realized that would work without creating a custom image myself.  I will try that and let you know how it goes.  (If that does work, seems it would be good to document, as it was non-obvious to me from the current docs.)

**Comment by JeremyRand** [comment]
It works!  The only non-obvious caveat is that I had to install Choco myself, whereas Cirrus's docker_builder images have it pre-installed.  (I just copied the install command from https://github.com/cirruslabs/vm-images/blob/01b4bd65881867b67b4c37ea25eec6ba1e725db8/googlecompute/windows_images.json#L36 into a script instruction at the start of the task, and copied the PATH command into the beginning of each of my script instructions that used `choco`.)

The main link that I'd suggest linking to in the docs is https://cloud.google.com/compute/docs/images/os-details , which contains the project and family codes.  That, along with the example task you pasted above, should be enough to make it obvious for newbies like me.
